src/command/cmd_defs.c

```diff
@@ -2325,12 +2325,12 @@ static const struct cmd_t command_defs[] = {
         CMD_DESC(
             "OMEMO commands to manage keys, and perform encryption during chat sessions.")
         CMD_ARGS(
-            { "gen",                     "Generate OMEMO crytographic materials for current account." },
+            { "gen",                     "Generate OMEMO cryptographic materials for current account." },
             { "start [<contact>]",       "Start an OMEMO session with contact, or current recipient if omitted." },
             { "end",                     "End the current OMEMO session." },
             { "log on|off",              "Enable or disable plaintext logging of OMEMO encrypted messages." },
             { "log redact",              "Log OMEMO encrypted messages, but replace the contents with [redacted]." },
-            { "fingerprint [<contact>]", "Show contact fingerprints, or current recipient if omitted." },
+            { "fingerprint [<contact>]", "Show contact's fingerprints, or current recipient's if omitted." },
```

Comment:
No english expert (as is evident for people who read my comments :D) but shouldn't it be "Show contacts fingerprints" without the "'"?